packages/aws/src/application/AWSCredentialsProvider.ts

```diff
@@ -23,9 +23,10 @@ export default class AWSCredentialsProvider {
           configLoader().AWS.authentication.options.proxyRoleName,
         )
       case 'AWS':
+        let partition = configLoader().AWS.IS_AWS_GLOBAL ? `aws`:`aws-cn`
```

Comment:
Do you know if these are the only two options? If not, I would suggest creating a map based on the region specified in my comment above and using that to set this partition value.